src/main/kotlin/no/nav/syfo/aktivitetskrav/api/DocumentComponentDTO.kt

```diff
@@ -7,23 +7,11 @@ data class DocumentComponentDTO(
     val texts: List<String>,
 )
 
-fun List<DocumentComponentDTO>.serialize(): String {
-    return buildString {
-        this@serialize.forEach { documentComponentDTO ->
-            documentComponentDTO.title?.let {
-                appendLine(it)
-            }
-            documentComponentDTO.texts.forEach {
-                appendLine(it)
-            }
-            appendLine()
-        }
-    }
-}
-
 enum class DocumentComponentType {
     HEADER_H1,
     HEADER_H2,
+    HEADER_H3,
```

Comment:
Kanskje lurt å legge til disse nye typene i isdialogmote også?

Reply:
La til info i [lappen](https://trello.com/c/Qav57B4S/1603-legg-til-nav-header-og-annen-oblig-info-i-brevmal-for-isdialogmotepdfgen) på det 🙌